README.md

```diff
@@ -57,3 +57,10 @@ npm run start:authentication-svc
 # start the authorization service
 npm run start:authorization-svc 
 ```
+
+To run those services locally, you need to pass 2 env vars like this (executed in modules/authentication-svc):
+
+```bash
+export PRIVATE_CERT_PEM_FILE_PATH=test_keys/private.pem
+export IAM_STORAGE_FILE_PATH=dist/authN_TempStorageFile
```

Comment:
Also include a troubleshoot section for the SSL issue:
![image](https://user-images.githubusercontent.com/1992108/194001693-3283f677-ec62-4ddd-9482-aa4a13be3673.png)

Details here:
https://github.com/auth0/node-jsonwebtoken/issues/826

Reply:
@Nikhilbabup Are all the integration tests passing?

Reply:
@koekiebox Yes all the integration test passing

Reply:
![Screenshot from 2022-10-05 16-24-26](https://user-images.githubusercontent.com/36840801/194044735-ac5d56ff-107d-46d2-a746-634f99d1ec30.png)

Reply:
Perfect.